e2e/cephfs_helper.go

```diff
@@ -62,11 +63,45 @@ func createCephfsStorageClass(
 	params map[string]string,
 ) error {
 	scPath := fmt.Sprintf("%s/%s", cephFSExamplePath, "storageclass.yaml")
-	sc, err := getStorageClass(scPath)
+	scValue, err := getStorageClass(scPath)
 	if err != nil {
 		return err
 	}
 
+	sc, err := updateStorageClassParameters(&scValue, params, enablePool, f)
+	if err != nil {
+		return err
+	}
+
+	return createStorageClass(c, sc)
+}
+
+func createCephfsStorageClassWaitForFirstConsumer(c kubernetes.Interface, f *framework.Framework,
+	enablePool bool,
+	params map[string]string) error {
+	scPath := fmt.Sprintf("%s/%s", cephFSExamplePath, "storageclass.yaml")
+	scValue, err := getStorageClass(scPath)
+	if err != nil {
+		return err
+	}
+
+	sc, err := updateStorageClassParameters(&scValue, params, enablePool, f)
+	if err != nil {
+		return err
+	}
+
+	// Set the volume binding mode to WaitForFirstConsumer
+	value := scv1.VolumeBindingWaitForFirstConsumer
+	sc.VolumeBindingMode = &value
+
+	return createStorageClass(c, sc)
+}
+
+func updateStorageClassParameters(sc *scv1.StorageClass, params map[string]string, enablePool bool, f *framework.Framework) (*scv1.StorageClass, error) {
```

Comment:
Why return the StorageClass? The passed `sc` is updated, just as the function name indicates. The returned `sc` is not new (or a copy). Just return an error only to prevent any confusion.